f2/src/forms/BusinessInfoForm.js

```diff
@@ -1,99 +1,133 @@
+    },
+    {
+      name: 'fiveHundredMore',
+      radioLabel: <Trans id="numberOfEmployee.500More" />,
+      radioName: 'numberOfEmployee.500More',
+      radioValue: 'numberOfEmployee.500More',
+    },
   ]
+
   return (
     <React.Fragment>
-      {false ? ( // mark ids for lingui
-        <div>
-          <Trans id="numberOfEmployee.1To99" />
-          <Trans id="numberOfEmployee.100To499" />
-          <Trans id="numberOfEmployee.500More" />
-        </div>
-      ) : null}
-      <Form
+      <Formik
         initialValues={businessInfo}
-        onSubmit={localOnSubmit}
-        render={({ handleSubmit }) => (
-          <Stack
-            as="form"
-            onSubmit={handleSubmit}
-            shouldWrapChildren
-            spacing={6}
-          >
-            <Field
-              name="nameOfBusiness"
-              label={<Trans id="businessPage.nameOfBusiness" />}
-              component={Input}
-            />
-
-            <Field
-              name="industry"
-              label={<Trans id="businessPage.industry" />}
-              helperText={<Trans id="businessPage.industryExample" />}
-              component={Input}
-            />
-
-            <Field
-              name="role"
-              label={<Trans id="businessPage.role" />}
-              helperText={<Trans id="businessPage.roleExample" />}
-              component={Input}
-            />
-            <FormArrayControl
-              name="numberOfEmployee"
-              label={<Trans id="numberOfEmployee.label" />}
-              helperText={<Trans id="numberOfEmployee.labelExample" />}
-            >
-              {numberOfEmployee.map((key) => {
-                return (
-                  <React.Fragment key={key}>
-                    <RadioAdapter name="numberOfEmployee" value={key}>
-                      {i18n._(key)}
-                    </RadioAdapter>
-                  </React.Fragment>
-                )
-              })}
-            </FormArrayControl>
-
-            <NextAndCancelButtons
-              next={<Trans id="businessInfoPage.nextPage" />}
-              button={<Trans id="businessInfoPage.nextButton" />}
-            />
-          </Stack>
+        onSubmit={(values) => {
+          props.onSubmit(values)
+        }}
+      >
+        {({ handleSubmit, handleChange, handleBlur }) => (
+          <Form onSubmit={handleSubmit}>
+            <Container>
+              <Row className="form-question">
+                <ErrorMessage name="businessInfo" component={Error} />
+              </Row>
+              <Row className="form-section">
+                <Field
+                  name="nameOfBusiness"
+                  label={<Trans id="businessPage.nameOfBusiness" />}
+                  component={Input}
+                  onChange={handleChange}
+                  onBlur={handleBlur}
+                  id={'name-of-business'}
+                />
+              </Row>
+              <Row className="form-section">
```

Comment:
can we remove the <Row> component to keep the three <Field> tighter?  right now, I feel the space between them is a little bit large.

Reply:
Good catch. I fixed it.